<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Unordered_map</class>
 <widget class="QMainWindow" name="Unordered_map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 0, 127);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_Umap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:7px;
font: 700 20pt &quot;System&quot;;
color:rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;System&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>unordered_map&lt;std::string,int&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Umap">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>150</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="do_Umap">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:7px;
font: 700 20pt &quot;System&quot;;
color:rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>do</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;System&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Key :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Umap">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color: rgb(255, 255, 255);
font: 700 20pt &quot;System&quot;;
color: rgb(0, 0, 255)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Umap">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color: rgb(255, 255, 255);
font: 700 20pt &quot;System&quot;;
color: rgb(0, 0, 255)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 20pt &quot;System&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>Data :</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>400</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="InfoSet">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>415</y>
      <width>231</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
